src/main/java/net/consensys/wittgenstein/core/NetworkLatency.java

```diff
@@ -143,6 +143,7 @@ public int getExtendedLatency(Node from, Node to, int delta) {
       Integer reg1 = regionPerCity.get(from.cityName);
       Integer reg2 = regionPerCity.get(to.cityName);
 
+
```

Comment:
Not a big deal but not really needed either :-)

Reply:
I will fix it